src/Symfony/Component/Validator/composer.json

```diff
@@ -44,6 +44,7 @@
         "doctrine/lexer": "<1.0.2",
         "phpunit/phpunit": "<5.4.3",
         "symfony/dependency-injection": "<4.4",
+        "symfony/expression-language": "<5.1",
```

Comment:
I don't think this conflicts here

Reply:
Will it be correct add here a check if the lint method exists? We can specify the required version in exception message.
https://github.com/symfony/symfony/blob/f6e0f0c3776e0748f93cec6083846e842c02dfbb/src/Symfony/Component/Validator/Constraints/ExpressionLanguageSyntaxValidator.php#L35-L46

Reply:
you are right!